docs/using.rst

```diff
@@ -412,33 +417,34 @@ Overriding the attribute protocol
 Subclasses which override the attribute-management methods provided by
 :class:`persistent.Persistent`, but must obey some constraints:
 
-:meth:`__getattribute__``
+
+``__getattribute__``
```

Comment:
It just runs sphinx with the warnings cranked up is all.

But that change is really unrelated, I'll revert.